flake8_django/checkers/model_dunder_str.py

```diff
@@ -28,6 +28,9 @@ def run(self, node):
         if not self.checker_applies(node):
             return
 
+        if self.is_abstract_model(node):
+            return
```

Comment:
@denizdogan I think we could make `self.is_abstract_model` check inside `checker_applies`:

```python
def checker_applies(self, node):
    if self.is_abstract_model(node):
        return False
    for base in node.bases:
        if self.is_model_name_lookup(base) or self.is_models_name_lookup_attribute(base):
            return True
    return False
```